data/map systems.txt

```diff
@@ -8533,6 +8654,73 @@ system Cinxia
 			distance 270
 			period 16.6355
 
+system Citadelle
+	pos -980.667 225.917
+	government Pirate
+	arrival 1240
+	habitable 1240
+	haze _menu/haze-33
+	link Fumalsamakah
+	asteroids "small rock" 28 1.404
+	asteroids "medium rock" 7 1.24
+	asteroids "large rock" 15 2.059
+	asteroids "small metal" 1 1.287
+	asteroids "medium metal" 27 1.217
+	asteroids "large metal" 3 1.661
+	minables iron 6 1.7
+	minables lead 7 2.9
+	trade Clothing 260
+	trade Electronics 882
+	trade Equipment 470
+	trade Food 260
+	trade "Heavy Metals" 832
+	trade Industrial 911
+	trade "Luxury Goods" 1190
+	trade Medical 879
+	trade Metal 236
+	trade Plastic 278
+	fleet "Small Southern Pirates" 1000
+	fleet "Large Southern Pirates" 2500
+	fleet "Small CCOR" 3800
+	fleet "Large CCOR" 4500
+	object
+		sprite star/k0
+		period 27.5
+	object
+		sprite star/a-dwarf
+		distance 102.423
+		period 27.3
+	object Bailey
+		sprite planet/lava0
+		distance 350
+		period 80
```

Comment:
Those are rookie numbers.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/26c6857c-5ac0-4635-8d06-d378d314c0fe)
```suggestion
		distance 192.4
		period 30.3149
```
And orbiting once every 30 days makes more sense for needing to consistently move the spaceport.

Reply:
I was only skeptical about the smaller binary star being too close with shorter distances like these; is that fine?

Reply:
Ignore that.